extension/jaegerremotesampling/README.md

```diff
@@ -24,7 +24,7 @@ Note that the port `14250` will clash with the Jaeger Receiver. When both are us
 
 Although this extension is derived from Jaeger, it can be used by any clients who can consume this standard, such as the [OpenTelemetry Java SDK](https://github.com/open-telemetry/opentelemetry-java/tree/v1.9.1/sdk-extensions/jaeger-remote-sampler).
 
```

Comment:
This README diff captures the essence of the PR (and reflects my intent to use the same config field/language rather than going into the nuanced differences)